brayns/common/types.h

```diff
@@ -308,7 +312,8 @@ enum class CameraType
     perspective,
     stereo,
     orthographic,
-    panoramic
+    panoramic,
+    clipped
```

Comment:
That's an implementation detail to me. Can't you put the planes in the scene and then do the glue between the scene and the camera for the planes inside the OSPray implementation?